src/data/grammar.json

```diff
@@ -1897,12 +1905,30 @@
         "Tutor for #_card2#",
         "Cast Pithing Needle naming #_card1#",
         "Discard #_card2# to Pack Rat",
-        "Make three copies of #_card1#",
+        "Make #count# copies of #_card1#",
         "Cast #_card1#, immediately say “Oops”, pick it back up, and cast #_card2#",
-        "Pitch #_card2# to #verbGerund.capitalizeAll# Shoal"
-    ],
+        "Pitch #_card2# to #verbGerund.capitalizeAll# Shoal",
+        "Hold #_card2# to your face and pretend to study the rules text",
+        "Drop #_card1# into your lap",
+        "Tap emphatically on #_card2# while staring your opponent straight in the eyes",
+        "Drop your cards on the table and give your opponent a puzzled smile",
+        "Imprint #_card2# on [_card3:Isochron Scepter,Spellbinder,Soul Foundry,Panoptic Mirror,Mimic Vat]#_card3#",
+        "Suspend #_card1# with Jhoira of the Ghitu",
+        "Tap exactly [_lands:one land,#count# lands]#_lands#",
+        "Retrieve #_card2# from your sideboard with #wishCard#",
+        "Flip the table",
+        "Play Three-Card Monte with #_card1#, #_card2#, and a morphed #realCard#",
+        "Toss #_card2# into the air and shrug"
+    ],
+    "inGraphicsCardMask": ["http://i.imgur.com/AKqp4qU.png"],
```

Comment:
I didn't work out yet how to host the images files locally, so they're still remote for now. But they should be local.